src/activities/Elsa.Activities.AzureServiceBus/Activities/SendAzureServiceBusMessage/SendAzureServiceBusTopicMessage.cs

```diff
@@ -22,6 +23,9 @@ public SendAzureServiceBusTopicMessage(ITopicMessageSenderFactory messageSenderF
 
         [ActivityInput] public string TopicName { get; set; } = default!;
         [ActivityInput] public object Message { get; set; } = default!;
+        
+        [ActivityInput(DefaultValue = true, Hint = "Allow the activity to send immediatly the message if true or wait for the first suspend. Usefull for RequestResponse pattern using Topic Message Received " )] 
+        public bool FireAndForget { get; set; } = true;
```

Comment:
As [you suggested on Discord](https://discord.com/channels/814605913783795763/814605914212270101/849946099367018517), let's move this to the "Advanced" category.